packages/flutter/test/material/checkbox_test.dart

```diff
@@ -67,11 +67,12 @@ void main() {
       ),
     ));
 
-    expect(tester.getSemantics(find.byType(Checkbox)), matchesSemantics(
+    expect(tester.getSemantics(find.byType(Focus)), matchesSemantics(
```

Comment:
Why the change from `Checkbox` to `Focus` here?

Reply:
Because that gets us down past the `RenderMouseRegion` so that it gets the `_CheckboxRenderObjectWidget` render object to query semantics from.  Since `Focus` doesn't have a render object associated with it, it defers to the `_CheckboxRenderObjectWidget`.